models/MOM6/model_mod.f90

```diff
@@ -220,25 +221,29 @@ end function get_model_size
 ! 0 unless there is some problem in computing the interpolation in
 ! which case a positive istatus should be returned.
 
-subroutine model_interpolate(state_handle, ens_size, location, qty, expected_obs, istatus)
+subroutine model_interpolate(state_handle, ens_size, location, qty_in, expected_obs, istatus)
 
 
 type(ensemble_type), intent(in) :: state_handle
 integer,             intent(in) :: ens_size
 type(location_type), intent(in) :: location
-integer,             intent(in) :: qty
+integer,             intent(in) :: qty_in
 real(r8),           intent(out) :: expected_obs(ens_size) !< array of interpolated values
 integer,            intent(out) :: istatus(ens_size)
 
+real(r8), parameter             :: concentration_to_ppt = 1000.0
```

Comment:
I am guessing that this parameter is to replace the 1000 in line 
```
382  expected_obs = expected_obs/1000.0_r8
```

so I'm going to go ahead and make that change

Reply:
done: f8856a624e7b4287f3d82dbbf7d98c891ecfe3a6